css/style.css

```diff
@@ -5,6 +5,7 @@
   padding: 0;
   box-sizing: border-box;
   font-family: "Roboto", sans-serif;
+  scrollbar-width: none; /*for Firefox */
```

Comment:
Appreciate that you have taken care of firefox from flipkart.

Reply:
Thank You!